subsys/caf/modules/Kconfig.ble_adv

```diff
@@ -35,6 +35,73 @@ config CAF_BLE_ADV_DIRECT_ADV
 config CAF_BLE_ADV_FAST_ADV
 	bool "Enable period of fast advertising"
 
+config CAF_BLE_ADV_CUSTOM_INT
```

Comment:
I think this option is not needed, because we provide defaults.

Reply:
Removing it would simplify the code too.

Reply:
Shouldn't we rely on `BT_GAP_ADV_FAST_INT...` that are included in gap.h? In future it may be difficult to track whether those value change. Although I don't think they ever will, I think it's cleaner to keep it that way, but I'm not really insisting on it. @MarekPieta what do you think?

Reply:
I think that it's not needed. The advertising intervals can in general be application-specific and may not rely on the definitions in GAP.